content/blog/azure-native-v3/index.md

```diff
@@ -0,0 +1,602 @@
+|----------|--------|-------|-----------|-------------------|------------------|
+| .NET     | 77MB | 15MB | 80.5% | 7m 52s | 47s |
+| JS/TS    | 494MB | 87MB | 82.4% | 166s | 25s |
+| Python   | 104.6 MB | 25MB | 76.3% | - | - |
+
+This transformation delivers immediate benefits that every developer will notice:
+
+- Faster IDE performance with reduced load times
+- More responsive language servers for better autocompletion and documentation
+- Reduced memory consumption for resource-constrained development environments
+- Significantly quicker package installation and project bootstrap times
+- Improved CI/CD pipeline performance with faster dependency installation
+
+## Additional Improvements
+
+Azure Native V3 also includes several other enhancements:
+
+- **Reorganized module structure** better aligned with Azure SDK conventions
+- **Improved resource naming** for clearer distinction between similar resources
+- **Enhanced handling of nested properties** to avoid accidental overwriting
+- **Better state management** with the removal of unnecessary `__inputs` fields
+
+## Understanding Azure Versioning
+
+The Pulumi Azure Native provider needs to manage two different versioning systems:
+
+1. **Pulumi's Semantic Versioning**: Our SDKs follow the traditional major.minor.patch format (e.g., 3.0.0)
+   - Patch versions (3.0.1) contain bug fixes only
+   - Minor versions (3.1.0) add new resources and non-breaking features
+   - Major versions (3.0.0) may contain breaking changes
+
+2. **Azure's Date-based API Versions**: Azure resources use date-based versions like "2024-01-01" or "2023-05-15-preview"
+   - Each Azure service (resource provider) has its own set of API versions
+   - Breaking changes can occur between different API versions
+   - Preview versions (with "-preview" suffix) contain newer features that may not be stable yet
+
+## Default API Versions
+
+Azure Native V3 includes refreshed default API versions across nearly all resources, ensuring you're automatically working with the latest stable Azure features and capabilities without any extra configuration. The new default versions unlock powerful new Azure functionality out of the box, such as:
+
+- EventGrid (now 2025-02-15): Support for advanced filtering, dead-lettering, and custom event schemas
+- MachineLearningServices (now 2024-10-01): Enhanced compute instance capabilities and new model deployment options
+- Storage (now 2024-01-01): Improved performance tiers and expanded security features
+- KeyVault (now 2024-03-01): Enhanced certificate management and expanded access policy options
+
+## Non-default API versions
+
+Most users (approximately 95% based on our Pulumi Cloud data) will be perfectly served by these default API versions. However, there are situations where you might need to use a specific API version.
+
+For these cases, Azure Native V3 provides two flexible approaches:
+
+**Option 1: Accessing Explicit API Versions via Local Packages**
```

Comment:
nit. This looks a little odd in the preview because it's smaller than the steps; we might want to use heading 3 for this (and the `option 2` line below) and use bold or heading 4 for the steps.